src/java.desktop/share/native/liblcms/LCMS.c

```diff
@@ -31,43 +31,9 @@
 #include "Trace.h"
 #include "Disposer.h"
 #include <lcms2.h>
+#include <lcms2_plugin.h>
 #include "jlong.h"
 
-
-#define ALIGNLONG(x) (((x)+3) & ~(3))         // Aligns to DWORD boundary
-
-#ifdef USE_BIG_ENDIAN
-#define AdjustEndianess32(a)
-#else
-
-static
-void AdjustEndianess32(cmsUInt8Number *pByte)
-{
-    cmsUInt8Number temp1;
```

Comment:
The logic of this method is the same as _cmsAdjustEndianess32 from the "lcms2_plugin.h"